src/supertux/menu/options_menu.cpp

```diff
@@ -179,6 +183,77 @@ OptionsMenu::OptionsMenu(bool complete) :
     resolutions.push_back(fullscreen_size_str);
   }
 
+  // Sound Volume
+  sound_volumes.clear();
```

Comment:
If we want to support arbitrary volumes, then we first need some form of slider or integer spinner menu component.